windows_events/subscriber.go

```diff
@@ -0,0 +1,124 @@
+package windows_events
+
+import "C"
```

Comment:
@nikhitashanker77 do we know why are importing C here? I don't see any usage for it in the code.

Reply:
Yes, I don't need it here I have removed it now.